swarm/tools/flow_studio_ui/fragments/10-header.html

```
@@ -21,11 +21,11 @@ <h1>Flow Studio</h1>
         <button id="mode-author" class="active" title="Show all paths, CLI commands, and technical details" aria-pressed="true" data-uiid="flow_studio.header.mode.author">Author</button>
         <button id="mode-operator" title="Simplified view focused on run status and decisions" aria-pressed="false" data-uiid="flow_studio.header.mode.operator">Operator</button>
       </div>
-      <button id="profile-badge" class="muted fs-status-button fs-profile-badge" title="Current swarm profile" data-uiid="flow_studio.header.profile">
+      <button id="profile-badge" class="muted fs-status-button fs-profile-badge" title="Current swarm profile" aria-label="Current swarm profile" data-uiid="flow_studio.header.profile">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)